frontends/web/src/routes/device/bitbox01/bitbox01.jsx

```diff
@@ -76,40 +71,15 @@ class Device extends Component {
     }
   }
 
-  componentDidUpdate(prevProps) {
-    if (this.props.deviceID !== prevProps.deviceID) {
-      this.onDevicesRegisteredChanged();
-    }
-  }
-
-  onDevicesRegisteredChanged = () => {
-    apiGet('devices/registered').then(devices => {
-      const deviceIDs = Object.keys(devices);
-      const deviceRegistered = deviceIDs.includes(this.getDeviceID());
-
-      this.setState({
-        deviceRegistered,
-        deviceStatus: null
-      }, () => {
-        // only if deviceRegistered or softwarekeystore
-        if (this.state.deviceRegistered) {
-          this.onDeviceStatusChanged();
-        }
-      });
-    });
-  };
-
   onDeviceStatusChanged = () => {
-    if (this.state.deviceRegistered) {
-      apiGet('devices/' + this.props.deviceID + '/status').then(deviceStatus => {
-        if (['seeded', 'initialized'].includes(deviceStatus)) {
-          this.context.setSidebarStatus('');
-        } else {
-          this.context.setSidebarStatus('forceHidden');
-        }
-        this.setState({ deviceStatus });
-      });
-    }
+    apiGet('devices/' + this.props.deviceID + '/status').then(deviceStatus => {
+      if (['seeded', 'initialized'].includes(deviceStatus)) {
+        this.context.setSidebarStatus('');
+      } else {
+        this.context.setSidebarStatus('forceHidden');
```

Comment:
This is the last place where forceHide is used I'll check if we can remove it (in a different pr)